internal/controller/nginx/config/base_http_config_template.go

```diff
@@ -48,6 +48,20 @@ server {
     }
 }
 
+{{- /* Define custom log format */ -}}
+{{- /* Access log directives for AccessLog. If path is "off" we disable logging. */ -}}
```

Comment:
nit: comment is incorrect. We could just remove it, I don't know if it's adding much value.